src/utils/formatPriceImpact.test.ts

```diff
@@ -0,0 +1,11 @@
+import { Percent } from "@uniswap/sdk-core"
+import formatPriceImpact from "./formatPriceImpact"
+
+describe('formatPriceImpact', () => {
+  it('formats positive price impact', () => {
```

Comment:
because there's theoretically no such thing as "positive price impact", I'd just leave a comment preceding this line:

```
  // While there's theoretically no such thing as "positive price impact", this can show up due to a bug in routing-api, so it's still tested for.
```